requirements/base.txt

```diff
@@ -5,28 +5,57 @@
 #    make upgrade
 #
 anyjson==0.3.3
+    # via -r requirements/base.in
```

Comment:
Any idea why the comments are on new lines, instead of being end-of-line comments for the package/line they address?

Notably, this is different that most/all other `requirements.txt` files I've seen..

Reply:
Updating it. Maybe I used a different version of `pip-compile` before.

Reply:
This is the new format caused by the `pip-tools==5.5.0`.